modules/collect/src/main/java/com/opengamma/strata/collect/MapStream.java

```diff
@@ -374,6 +374,26 @@ private MapStream(Stream<Map.Entry<K, V>> underlying) {
     return underlying.map(e -> mapper.apply(e.getKey(), e.getValue()));
   }
 
+  /**
+   * Transforms the entries in the stream to doubles by applying a mapper function to each key and value.
+   *
+   * @param mapper a mapper function whose return values are doubles included in the new stream
```

Comment:
double space after param names in Strata, and not sure whether you need to specify that it's a double
```suggestion
   * @param mapper  a mapper function whose return values are included in the new stream
```